bats_ai/core/management/commands/importRecordings.py

```diff
@@ -0,0 +1,202 @@
+import logging
+from pathlib import Path
+
+from django.contrib.auth.models import User
+from django.contrib.gis.geos import Point
+from django.core.files import File
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+from bats_ai.core.models import Recording
+from bats_ai.core.utils.guano_utils import extract_guano_metadata
+from bats_ai.tasks.tasks import recording_compute_spectrogram
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = 'Import WAV files from a directory, extract GUANO metadata, and create recordings'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'directory',
+            type=str,
+            help='Directory path containing WAV files to import',
+        )
+        parser.add_argument(
+            '--owner',
+            type=str,
+            help='Username of the owner for the recordings (defaults to first superuser)',
+        )
+        parser.add_argument(
+            '--public',
+            action='store_true',
+            help='Make imported recordings public',
+        )
+        parser.add_argument(
+            '--limit',
+            type=int,
+            help='Limit the number of WAV files to import (useful for testing)',
+        )
+
+    def handle(self, *args, **options):
+        directory_path = Path(options['directory'])
+        owner_username = options.get('owner')
+        is_public = options.get('public', False)
+        limit = options.get('limit')
+
+        # Validate directory
+        if not directory_path.exists():
+            self.stdout.write(self.style.ERROR(f'Directory does not exist: {directory_path}'))
+            return
+
+        if not directory_path.is_dir():
+            self.stdout.write(self.style.ERROR(f'Path is not a directory: {directory_path}'))
+            return
+
+        # Get or find owner
+        if owner_username:
+            try:
+                owner = User.objects.get(username=owner_username)
+            except User.DoesNotExist:
+                self.stdout.write(self.style.ERROR(f'User not found: {owner_username}'))
+                return
+        else:
+            # Default to first superuser
+            owner = User.objects.filter(is_superuser=True).first()
+            if not owner:
+                self.stdout.write(
+                    self.style.ERROR(
+                        'No superuser found. Please specify --owner or create a superuser.'
+                    )
+                )
+                return
+            self.stdout.write(self.style.WARNING(f'Using default owner: {owner.username}'))
+
+        # Find all WAV files
+        wav_files = list(directory_path.rglob('*.wav'))
+        wav_files.extend(directory_path.rglob('*.WAV'))
```

Comment:
```suggestion
        # Find all WAV files
        wav_files = list(directory_path.rglob('*.wav', case_sensitive=False))
```
Looks like if you wanted to you could use the `case_sensitive` parameter of `rglob`\, although I suppose this would capture more than just `.wav` and `.WAV`.

https://docs.python.org/3/library/pathlib.html#pathlib.Path.rglob